website/src/pages/download.md

```diff
@@ -8,21 +8,27 @@ The official Apache OpenDAL (incubating) releases are as source artifacts.
-| 0.38.1-incubating | [tarball](https://dlcdn.apache.org/incubator/opendal/0.38.1/apache-opendal-incubating-0.38.1-src.tar.gz) | [asc](https://dlcdn.apache.org/incubator/opendal/0.38.1/apache-opendal-incubating-0.38.1-src.tar.gz.asc) | [sha512](https://dlcdn.apache.org/incubator/opendal/0.38.1/apache-opendal-incubating-0.38.1-src.tar.gz.sha512) |
-| 0.38.0-incubating | [tarball](https://dlcdn.apache.org/incubator/opendal/0.38.0/apache-opendal-incubating-0.38.0-src.tar.gz) | [asc](https://dlcdn.apache.org/incubator/opendal/0.38.0/apache-opendal-incubating-0.38.0-src.tar.gz.asc) | [sha512](https://dlcdn.apache.org/incubator/opendal/0.38.0/apache-opendal-incubating-0.38.0-src.tar.gz.sha512) |
-| 0.37.0-incubating | [tarball](https://dlcdn.apache.org/incubator/opendal/0.37.0/apache-opendal-incubating-0.37.0-src.tar.gz) | [asc](https://dlcdn.apache.org/incubator/opendal/0.37.0/apache-opendal-incubating-0.37.0-src.tar.gz.asc) | [sha512](https://dlcdn.apache.org/incubator/opendal/0.37.0/apache-opendal-incubating-0.37.0-src.tar.gz.sha512) |
-| 0.36.0-incubating | [tarball](https://dlcdn.apache.org/incubator/opendal/0.36.0/apache-opendal-incubating-0.36.0-src.tar.gz) | [asc](https://dlcdn.apache.org/incubator/opendal/0.36.0/apache-opendal-incubating-0.36.0-src.tar.gz.asc) | [sha512](https://dlcdn.apache.org/incubator/opendal/0.36.0/apache-opendal-incubating-0.36.0-src.tar.gz.sha512) |
-
-For older releases, please check the [archive](https://dlcdn.apache.org/incubator/opendal/).
+The last source release is [0.44.1-incubating](
+https://www.apache.org/dyn/closer.lua/incubator/opendal/0.44.1/)([asc](https://downloads.apache.org/incubator/opendal/0.44.1/apache-opendal-incubating-0.44.1-src.tar.gz.asc),
+[sha512](https://downloads.apache.org/incubator/opendal/0.44.1/apache-opendal-incubating-0.44.1-src.tar.gz.sha512)).
+
+### Version List
+
+| Name              | Archive                                                                                                      | Signature                                                                                                    | Checksum                                                                                                           |
+|-------------------|--------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------|
+| 0.44.1-incubating | [tarball](https://downloads.apache.org/incubator/opendal/0.44.1/apache-opendal-incubating-0.44.1-src.tar.gz) | [asc](https://downloads.apache.org/incubator/opendal/0.44.1/apache-opendal-incubating-0.44.1-src.tar.gz.asc) | [sha512](https://downloads.apache.org/incubator/opendal/0.44.1/apache-opendal-incubating-0.44.1-src.tar.gz.sha512) |
```

Comment:
It says the tarball MUST use https://www.apache.org/dyn/closer.lua.

Please check https://infra.apache.org/release-download-pages.html#download-page:

* must have at least one link to the current release. This link must use the closer.lua utility. For example: https://www.apache.org/dyn/closer.lua/PROJECT/VERSION/SOURCE-RELEASE. (Note: the mirrors.cgi and closer.cgi scripts have been deprecated. Calls to them redirect to closer.lua.)
* must have a link to the checksum for the current release. These links must use direct links to the Apache distribution server. For example: https://downloads.apache.org/PROJECT/VERSION/CHECKSUM.

Reply:
Also, we can retain the lastest release only.

Reply:
👀I only provided the last release with closer.lua.
Since it will add a extra step to download, I didn't apply it for all tarballs.

Reply:
See the suggestion below. With `?action=download` the script can trigger a download. While it doesn't seem to work with `curl -O ...`, it works with `wget -O ...`.